roofit/roofitcore/src/TestStatistics/LikelihoodGradientJob.cxx

```diff
@@ -37,9 +38,26 @@ LikelihoodGradientJob::LikelihoodGradientJob(std::shared_ptr<RooAbsL> likelihood
    minuit_internal_x_.reserve(N_dim);
 }
 
+LikelihoodGradientJob::LikelihoodGradientJob(std::shared_ptr<RooAbsL> likelihood,
+                                             std::shared_ptr<WrapperCalculationCleanFlags> calculation_is_clean,
+                                             std::size_t N_dim, RooMinimizer *minimizer,
+                                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets,
+                                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets_save)
+   : LikelihoodGradientWrapper(std::move(likelihood), std::move(calculation_is_clean), N_dim, minimizer, std::move(offsets), std::move(offsets_save)),
+     grad_(N_dim)
+{
+   // Note to future maintainers: take care when storing the minimizer_fcn pointer. The
```

Comment:
As of https://github.com/root-project/root/pull/12348, this comment is not true anymore. The `RooAbsMinimizerFcn` subclasses are now never cloned. In fact they can't even be cloned because the copy constructors are deleted.

I guess that means you can remove this comment here and also in other places? And probably you can simplify your code because you don't need to work around the `MinuitFcnGrad` being cloned?

Reply:
Ah, I overlooked this comment. I accidentally almost already addressed it by removing all cloning and copying support. The only thing left was to remove the comments, which I now did as well (amended the previous commit).